src/Curators/Events/NewsArticleAboutEventAddedJSONDeserializer.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace CultuurNet\UDB3\Silex\Curators\Events;
+namespace CultuurNet\UDB3\Curators\Events;
```

Comment:
The usage of this class (and others changed in this commits) should also be updated in https://github.com/cultuurnet/udb3-silex/blob/master/app/Curators/CuratorsServiceProvider.php